### PR TITLE
Remove legacy password schemes

### PR DESCRIPTION
**Pull request by sean-hayes**
Also incorporated:

i. as my email hinted, the user "can" method call allows for a missing "thing"
i. added FOUC support in the default theme.  This hides the flash of unstyled content on entities that explicitly enable it.

### CODE REVIEW

**Review by hlainchb — COMMENTED** [review]
Thanks for the PR.  These files are generated from src files.  Would you be interested in making the changes there?

**Comment by hlainchb** [comment]
Is .can being called somewhere without a thing?  I originally had that use of .can meaning whether or not it could run an app... but then I decided it was better to have a separate call to take care of that use case which is now called .can_run(appname). ... I may have missed making that change to a place where the old version was used.

File: zoom/users.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def can_run(self, app_name):
         """test if user can run an app"""
         return self.is_admin or self.is_developer or app_name in self.apps
 
-    def can(self, action, thing):
+    def can(self, action, thing=None):
```
Comment (sean-hayes) [inline zoom/users.py]: confirmed, you can undo this change at this line - I symlinked in the recent apps and this issue disappeared

**Comment by sean-hayes** [comment]
I highlighted the line you can undo.  Alternatively, let me know and I can resubmit.
